### PR TITLE
ROC-6954:Incorrect order for generate order fields

### PR DESCRIPTION
**Pull request by SanchitaMoj**
**Confirm that:**

- [ ] *VERSION* has been bumped (only for definition file changes)
- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)

### Change description ###

**Does this PR introduce a breaking change?**

```
- [ ] Yes
- [ ] No
```

### CODE REVIEW

File: definition/data/sheets/CaseEventToFields.json [hunk 1]
```diff
@@ -4,11 +4,13 @@
...
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "eyewitnessUploadForParty", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 5, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "docUploadForParty", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 6, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "eyewitnessUploadDeadline", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 2, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "eyewitnessUploadForParty", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 3, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
```
Comment (sabahirfan) [inline definition/data/sheets/CaseEventToFields.json]: shouldn't we update the `PageFieldDisplayOrder` for these above two fields?

File: definition/data/sheets/CaseEventToFields.json [hunk 2]
```diff
@@ -28,11 +30,11 @@
...
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "docUploadDeadline", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 5, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "extraDocUploadList", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 2, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "docUploadDeadline", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 3, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ActionReviewComments", "CaseFieldID": "docUploadForParty", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 6, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
```
Comment (sabahirfan) [inline definition/data/sheets/CaseEventToFields.json]: to make these items in sequence in this file as well, should't we move this after `eyewitnessUploadForParty`?

File: definition/data/sheets/CaseEventToFields.json (hunk 1 above)
Comment (SanchitaMoj) [reply definition/data/sheets/CaseEventToFields.json]: Missed deleting them, cleaned now

File: definition/data/sheets/CaseEventToFields.json (hunk 2 above)
Comment (SanchitaMoj) [reply definition/data/sheets/CaseEventToFields.json]: yes, done

**Pull request merged**